scripts/build-pdfs.js

```diff
@@ -1,13 +1,14 @@
 const fs = require('fs');
-const path = require('path');
 
 const PDFDocument = require('pdfkit');
 const SVGtoPDF = require('svg-to-pdfkit');
 const simpleIcons = require('simple-icons');
 
+const { getIconSlug } = require('./utils.js');
+
 for (const title in simpleIcons) {
-  const icon = simpleIcons.get(title);
-  const iconSlug = icon.slug || titleToSlug(icon.title);
+  const icon = simpleIcons.Get(title);
+  const iconSlug = getIconSlug(icon);
```

Comment:
This can be just `icon.slug` - which is also why the previous version of this script didn't fail. We don't need a copy of the utilities file in this repository.